src/lib/api.ts

```diff
@@ -243,7 +413,69 @@ export async function getCategories() {
 }
 
 export async function getGroupExpenses(groupId: string) {
+  const now: number = Math.floor((new Date()).getTime() / 1000)
```

Comment:
This calculation `Math.floor(date.getTime() / 1000` appears in a few places, it would probably improve readability to factor this into a separate function (and the inverse `*1000` calculation) since it's not super obvious what the calculation is for, I'm guessing converting a JS timestamp from milliseconds to seconds.

Reply:
Yes. Done.